recipe/install_python.sh

```diff
@@ -1,10 +1,34 @@
 #!/bin/bash
 
-pushd swig/python
+set -xe
 
-$PYTHON -m pip install --no-deps --ignore-installed . \
-        --global-option build_ext \
-        --global-option "-I$INCLUDE_PATH" \
-        --global-option "-L$LIBRARY_PATH"
+# now re-configure with BUILD_PYTHON_BINDINGS:BOOL=ON
+
+mkdir pybuild_${PKG_HASH}
+pushd pybuild_${PKG_HASH}
+
+cmake -G "Unix Makefiles" \
+      ${CMAKE_ARGS} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DBUILD_SHARED_LIBS=ON \
+      -DPython_EXECUTABLE="$PYTHON" \
+      -DBUILD_PYTHON_BINDINGS:BOOL=ON \
+      -DBUILD_JAVA_BINDINGS:BOOL=OFF \
+      -DBUILD_CSHARP_BINDINGS:BOOL=OFF \
+      ${SRC_DIR} || (cat CMakeFiles/CMakeError.log;false)
+
+
+cd swig/python
+
```

Comment:
"won't have been generated *in the build directory*"

Reply:
Thanks for the hint, @rouault. Would it make sense to do the SWIG_REGENERATE thing instead of the `cmake --build`?

Reply:
> Would it make sense to do the SWIG_REGENERATE thing instead of the `cmake --build`?

No, SWIG_RENEGERATE swig will not help if you don't run the python_wheel target. 
But I missed that python_wheel does build libgdal as a dependency.
OK, I'm proposing https://github.com/OSGeo/gdal/pull/6704 which adds a python_generated_files target that should do what you want, and doesn't require building libgdal. But you will need to provide to setup.py the path to the GDAL include and library, because by default it will point to in-tree paths/files that don't exist if you just run this python_generated_files target.